libs/designsystem/progress-circle/src/progress-circle-ring.component.ts

```diff
@@ -39,7 +30,18 @@ export class ProgressCircleRingComponent {
     return this._centerRadius * 2 * Math.PI;
   }
 
-  private calculateOffset(value: number): number {
-    return this._centerCircumference - this._centerCircumference * (value / 100);
+  get _progress(): number {
+    const valueWithinBounds = this.value < this.upperBound || this.value > 99;
+    const _value = valueWithinBounds ? this.value : this.upperBound;
+
+    return (_value * this._centerCircumference) / 100;
```

Comment:
Consider readability, ie. as in the original `calculateOffset` method - the progress is a percentage of 100 times the circumference 
```suggestion
    return this._centerCircumference * (_value / 100);
```
OR even more explicit:
```suggestion
    const progressPercentage = _value / 100;
    return this._centerCircumference * progressPercentage;
```